File: presentation/src/main/java/by/grodno/vasili/githubsimpleapp/GithubSimpleApplication.java
package by.grodno.vasili.githubsimpleapp;

import android.app.Application;

import timber.log.Timber;

public class GithubSimpleApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        initTimber();
    }

    private void initTimber() {
        if (BuildConfig.DEBUG) {
            Timber.plant(new Timber.DebugTree());
        }
    }
}

File: presentation/src/main/java/by/grodno/vasili/githubsimpleapp/GithubSimpleApplication.kt
package by.grodno.vasili.githubsimpleapp

import android.app.Application
import timber.log.Timber
import timber.log.Timber.DebugTree

class GithubSimpleApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        initTimber()
    }

    private fun initTimber() {
        if (BuildConfig.DEBUG) {
            Timber.plant(DebugTree())
        }
    }
}
